pyodata/v2/model.py

```diff
@@ -2072,13 +2083,24 @@ def parse(metadata_xml):
             mdf = io.BytesIO(metadata_xml)
         else:
             raise TypeError('Expected bytes or str type on metadata_xml, got : {0}'.format(type(metadata_xml)))
-        # the first child element has name 'Edmx'
-        edmx = etree.parse(mdf)
+
+        edmx: etree._ElementTree = etree.parse(mdf)
```

Comment:
Woohoo, what does this syntax do?

Reply:
It mainly aids the autocomplete as the real signature isn't inherently known as lxml is not directly written in python.

Reply:
Can we get rid of it?

Reply:
done